projects/metis/src/app/dataset/workflow/workflow-header/workflow-header.component.ts

```diff
@@ -41,6 +41,15 @@ export class WorkflowHeaderComponent implements AfterViewInit {
       ctrl.setValue(!ctrl.value);
       this.workflowForm.markAsDirty();
     }
+    if (plugin === 'pluginHARVEST') {
+      const ctrl = this.workflowForm.get('pluginType') as FormControl;
```

Comment:
You must be able to find this string value 'pluginType' in the workflow settings somewhere, right? It should be a constant or otherwise obtainable from the workflow config?

Reply:
It's defined [here](https://github.com/europeana/metis-ui/blob/develop/projects/metis/src/app/_models/workflow-form.ts#L23-L33) as a type, and types "cannot be used as a value".

Reply:
I understand, but isn't it better in this case to have `this.workflowForm.get(ParameterFieldName.pluginType)`? Or wouldn't that work for some reason?